flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/stream/StreamExecTemporalJoin.scala

```diff
@@ -241,7 +233,7 @@ class StreamExecTemporalJoinToCoProcessTranslator private (
     val minRetentionTime = tableConfig.getMinIdleStateRetentionTime
     val maxRetentionTime = tableConfig.getMaxIdleStateRetentionTime
     joinType match {
-      case JoinRelType.INNER =>
+      case JoinRelType.INNER | JoinRelType.LEFT =>
```

Comment:
Does runtime support left join? this change should be covered by at least one IT case

Reply:
Yes, runtime will support left join in next PR